### DISCUSSION TITLE
How to get the full list of comfyUI API?

### DISCUSSION THREAD

**Discussion by yue916** [comfyanonymous/ComfyUI#4918] [Q&A]
i want to get the full list of  comfyUI API that can be invoke,thanks for any help

**Comment by ltdrdata** [comment]
You can look at the endpoints in the core that are set with the `@routes.???` annotation.
https://github.com/comfyanonymous/ComfyUI/blob/369a6dd2c499a02b42496bab6286e34e9350c740/server.py#L185
